lib/src/view/analysis/analysis_position_choice_screen.dart

```diff
@@ -85,7 +85,7 @@ class _BodyState extends State<_Body> {
               child: AdaptiveTextField(
                 maxLines: 500,
                 placeholder:
-                    '${context.l10n.pasteTheFenStringHere}\n\n${context.l10n.pasteThePgnStringHere}\n\nLeave empty for initial position',
+                    '${context.l10n.pasteTheFenStringHere}\n\n${context.l10n.pasteThePgnStringHere}\n\n${context.l10n.mobileLoadPositionLeaveEmpty}',
```

Comment:
I'd say let's not translate that one for now. I don't think this screen will stay like this. And it would be better with a dedicated message.